### PR TITLE
build: update to rxjs@5.5.0

### PR DESCRIPTION
**Pull request by jasonaden**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "core-js": "^2.4.1",
     "reflect-metadata": "^0.1.3",
-    "rxjs": "5.x",
+    "rxjs": "5.5.0-beta.4",
```
Comment (IgorMinar) [inline package.json]: beta.7 is the latest

File: yarn.lock [hunk 2]
```diff
@@ -111,18 +111,14 @@
   version "2.2.22-alpha"
   resolved "https://registry.yarnpkg.com/@types/jasmine/-/jasmine-2.2.22-alpha.tgz#eecaee43fe42ef6b5cfefad1bcc370c433f485bf"
 
-"@types/node@*":
-  version "8.0.28"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-8.0.28.tgz#86206716f8d9251cf41692e384264cbd7058ad60"
+"@types/node@*", "@types/node@7.0.18":
+  version "7.0.18"
+  resolved "https://registry.yarnpkg.com/@types/node/-/node-7.0.18.tgz#cd67f27d3dc0cfb746f0bdd5e086c4c5d55be173"
```
Comment (IgorMinar) [inline yarn.lock]: this and many other changes in the lock file should not be here. you should be updating just the rxjs version and nothing else.

**Review by IgorMinar — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (jasonaden) [reply yarn.lock]: So I'm doing the upgrade with a command like this:

`yarn upgrade rxjs@5.5.0-beta.7`

After doing this, I get one line updated in package.json, and tons of updates in the yarn.lock file. Looks like it's due to Rx's dependencies.

Comment (IgorMinar) [reply yarn.lock]: are you using yarn 1.0.2?

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**